warpscript/build.gradle

```diff
@@ -46,6 +46,7 @@ jar {
     include 'io/warp10/WarpURLDecoder.class'
     include 'io/warp10/DoubleUtils.class'
     include 'io/warp10/FloatUtils.class'
+    include 'io/warp10/ListWrapper.class'
```

Comment:
Does this include the ArrayItr class too?

Reply:
Since the inner class `ArrayItr` is public, it is in the ListWrapper.class file.